### PR TITLE
Implemented a pseudoQueue

### PR DESCRIPTION
**Pull request by UreaLaden**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code-challenges/javaChallenges/lib/src/test/java/javaChallenges/PseudoQueueTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    Stack<Object> temp = new Stack<>();
+    PseudoQueue<Object> pQTest = new PseudoQueue<>(main,temp);
+    pQTest.enqueue(4);
+    pQTest.enqueue(3);
+    pQTest.enqueue(2); // {2} -> {3} -> {4} -> NULL
+    pQTest.dequeue(); // {2} -> {3} -> NULL
+    System.out.println(pQTest.getMain().toString());
+  }
+
+  @Test public void errorTest(){
```
Comment (Mattpet26) [inline code-challenges/javaChallenges/lib/src/test/java/javaChallenges/PseudoQueueTest.java:37]: Great work! Test the methods you wrote AND base case. What happens if the stack is empty and I try X or Y.
